src/tdh.ts

```diff
@@ -168,7 +168,7 @@ export const findTDH = async (lastTDHCalc: Date) => {
         areEqualAddresses(t.contract, GRADIENT_CONTRACT)
       );
 
-      ADJUSTED_NFTS.map((nft) => {
+      ADJUSTED_NFTS.forEach((nft) => {
         let tokenTransactions: Transaction[] = [];
         if (areEqualAddresses(nft.contract, MEMES_CONTRACT)) {
           tokenTransactions = [...memesTransactions].filter(
```

Comment:
just fyi. this seems like a pointless loop since tokenTransactions is an unused variable

Reply:
loop isn't pointless, just the declaration and assigning of 'tokenTransactions'